src/main/java/uk/gov/hmcts/ccd/v2/external/controller/TestingSupportController.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.ccd.v2.external.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.ccd.domain.model.casedeletion.CaseLinksResource;
+import uk.gov.hmcts.ccd.domain.service.casedeletion.CaseLinkService;
+import uk.gov.hmcts.ccd.v2.V2;
+
+@RestController
+@RequestMapping(path = "/testing-support")
+@ConditionalOnProperty(value = "testing.support.endpoint.enabled", havingValue = "true")
+public class TestingSupportController {
+
+    private CaseLinkService caseLinkService;
+
+    @Autowired
```

Comment:
`@Inject`?

Reply:
Done in [fe6c52f](https://github.com/hmcts/ccd-data-store-api/pull/1719/commits/fe6c52f4bc3d7709a1a740daa2b74ad122b5a4a1)